test/decoder/DecUT_PredMv.cpp

```diff
@@ -249,6 +255,9 @@ TEST (PredMvTest, PredInter8x16Mv) {
   const int32_t kiRandTime = 100;
   bool bOK = true;
 
+  memset(&sWelsMvPred, 0, sizeof(SWelsMvPred));
+  memset(&sAncMvPred, 0, sizeof(SAnchorMvPred));
+
```

Comment:
How about the more type safe `SWelsMvPred sWelsMvPred = {}` instead?

Reply:
Can I know why this is more type safe, given that we know the definition of the structure?

Reply:
Because you can accidentally mix up the sizes e.g.:
```c
  memset(&sWelsMvPred, 0, sizeof(SAnchorMvPred));
  memset(&sAncMvPred, 0, sizeof(SWelsMvPred));
  ```